VisioShape.ps1

```diff
@@ -130,3 +130,202 @@ Function Get-VisioShape{
+                    HPE 36U G2 Adv Rack Front
+                    HPE 36U G2 Adv Rack Rear
+                    HPE 42U 800mm G2 Adv Rack Front
+                    HPE 42U 800mm G2 Adv Rack Rear
+                    HPE 42U 800mm G2 Ent Rack Front
+                    HPE 42U 800mm G2 Ent Rack Rear
+                    HPE 42U G2 Adv Rack Front
+                    HPE 42U G2 Adv Rack Rear
+                    HPE 42U G2 Ent Rack Front
+                    HPE 42U G2 Ent Rack Rear
+                    HPE 48U 800mm G2 Adv Rack Front
+                    HPE 48U 800mm G2 Adv Rack Rear
+                    HPE 48U 800mm G2 Ent Rack Front
+                    HPE 48U 800mm G2 Ent Rack Rear
+                    HPE 48U G2 Adv Rack Front
+                    HPE 48U G2 Adv Rack Rear
+                    HPE 48U G2 Ent Rack Front
+                    HPE 48U G2 Ent Rack Rear
+                    50U Ent. Rack
+
+            Dell shapes from the Dell-Racks set on VisioCafe - https://www.visiocafe.com/downloads/dell/Dell-Racks.zip
+
+                    2420 Rack Frame
+                    4220 Rack Frame
+                    4220D Rack Frame
+                    4220W Rack Frame
+                    4820 Rack Frame
+                    4820D Rack Frame
+                    4820W Rack Frame
+
+            IBM shapes from the IBM-Racks set on VisioCafe - https://www.visiocafe.com/downloads/ibm/IBM-Common.zip
+
+                    7014-S11 Rack
+                    7014-S25 Rack
+                    7014-S00 Rack
+                    7014-T00 Rack
+                    7014-T42 Rack
+
+            Cisco shapes from the Cisco R-Series set on Cisco.com - https://www.cisco.com/c/dam/assets/prod/visio/visio/racks-cisco-r-series.zip
+
+                    RACK2-UCS Front
+                    RACK2-UCS Rear
+                    RACK2-UCS2 Front
+                    RACK2-UCS2 Rear
+
+        .PARAMETER FirstU
+        The first (lowest) U space in which to place the new rack equipment shape.
+
+        .PARAMETER X
+        The X position used for initial new rack equipment shape placement (in inches) prior to connecting it to the existing rack shape. If not provided the default X position = 1
+
+        .PARAMETER Y
+        The Y position used for initial new rack equipment shape placement (in inches) prior to connecting it to the existing rack shape. If not provided the default Y position = 0.25
+
+        .OUTPUTS
+        Visio.Shape
+
+        .EXAMPLE
+        New-VisioRackShape -Master HPEDL180Gen108LFF -Label Server1 -RackLabel Rack01 -RackVendor HPE -FirstU 1
+#>
+Function New-VisioRackShape{
+    [CmdletBinding(SupportsShouldProcess=$True)]
+    Param (
+	    [Parameter(Mandatory=$true)]$Master,
+	    [Parameter(Mandatory=$true)]$Label,
+	    [Parameter(Mandatory=$false)]$Name,
+	    [Parameter(Mandatory=$true)]$RackLabel,
+	    [Parameter(Mandatory=$true)]$RackVendor,
+	    [Parameter(Mandatory=$true)]$FirstU,
+		[Parameter(Mandatory=$false)]$X,

```

Comment:
Should these be indented?

Reply:
No idea why `[Parameter(Mandatory=$false)]$X,` and `[Parameter(Mandatory=$false)]$Y` are showing as 1 level further indented than the other 6 parameters, they are not when I view the code in VSC.

![VisioBot3000 PR78 New-VisioRackShape Indented Parameter Block from VSC](https://user-images.githubusercontent.com/13692287/135587229-65cfd686-7e53-4c8f-8ec9-9a1965fb84ef.PNG)